scripts/suite_objects.py

```diff
@@ -1727,11 +1731,13 @@ def write(self, outfile, errcode, errmsg, indent):
         # 
         # Write the scheme call.
         #
-        stmt = 'call {}({})'
-        outfile.write('',indent+1)
-        outfile.write('! Call scheme', indent+1)
-        outfile.write(stmt.format(self.subroutine_name, my_args), indent+1)
-        outfile.write('',indent+1)
+        if not self._no_run_phase:
```

Comment:
This is awkward. Maybe I redo logic to be "has_run_phase" instead of "no_run_phase"
Thoughts?

Reply:
I like `has_run_phase` much better!

Reply:
Same here. I will review after you made the change